docs/metrics.md

```diff
@@ -0,0 +1,7 @@
+# Virtual Machine Import metrics
+
+Virtual Machine Import uses Prometheus for metrics reporting. The metrics can be used for real-time monitoring. Virtual Machine Import does not persist its metrics, if a member restarts, the metrics will be reset.
+
+| Name                      | Description                                              | Type    |
+|---------------------------|----------------------------------------------------------|---------|
+| kubevirt_vmimport | The total number of Virtual Machine imports.               | Counter |
```

Comment:
I think it would be valuable to track vm import result like number how many import failed.

Reply:
Sure, will add.

Reply:
I've added label `result` with `successful` and `failed`:
<img width="1667" alt="Screenshot 2020-07-08 at 13 34 41" src="https://user-images.githubusercontent.com/3384302/86914141-1265d800-c120-11ea-9504-e9158ca2d5d0.png">